Cabal/Distribution/Simple/Configure.hs

```diff
@@ -413,20 +445,28 @@ configure (pkg_descr0', pbi) cfg = do
                 allConstraints
                 (dependencySatisfiable
                     (fromFlagOrDefault False (configExactConfiguration cfg))
+                    (packageVersion pkg_descr0)
                     installedPackageSet
                     internalPackageSet
                     requiredDepsMap)
                 comp
                 compPlatform
                 pkg_descr0
 
+    debug verbosity $ "Finalized package description:\n"
+                  ++ showPackageDescription pkg_descr
```

Comment:
Do we want to keep this or was this only during hacking? I ask since it's quite a lot of text to dump.